tests/foreman/installer/test_installer.py

```diff
@@ -1390,6 +1391,40 @@ def test_installer_options_and_sections(filter):
     assert previous == current, msg
 
 
+@pytest.mark.e2e
+@pytest.mark.tier1
+@pytest.mark.parametrize("sat_ready_rhel", [settings.server.version.rhel_version], indirect=True)
+def test_satellite_installation(sat_ready_rhel):
+    """Run a basic Satellite installation
+
+    :id: bbab30a6-6861-494f-96dd-23b883c2c906
+
+    :steps:
+        1. Get a RHEL host
+        2. Configure repos
+        3. Install satellite
```

Comment:
non-blocking:
```suggestion
        3. Run satellite installer
```

Reply:
Would suggest something like, Enable satellite module, install satellite and run satellite-installer